apps/site/src/views/Resources/components/ResourceCard/ResourceCard.module.scss

```diff
@@ -0,0 +1,55 @@
+.group {
+	margin-top: 50px;
+	width: 344px;
+	position: relative;
+	box-shadow: 0 6px 5px -2px rgba(gray, 0.5);
+}
+
+.tape {
+	background-image: url("~@/assets/images/tape.svg");
+	width: 238px;
+	height: 60px;
+	position: absolute;
+	top: -5%;
+}
+
+.container {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	gap: 10px;
+	height: 400px;
+}
+
+.tag {
+	background-image: url("~@/assets/images/resource-link.svg");
+	background-size: 100% 100%;
+	background-repeat: no-repeat;
+	background-position: left;
+	display: inline-block;
+	padding: 1rem;
+	padding-right: 3rem;
+	margin-left: -6px;
+	margin-bottom: 0.5rem;
+	transition:
+		filter 0.13s ease,
+		transform 0.13s ease,
+		filter 0.13s ease;
+
+	color: black;
+	text-decoration: none;
+	font-weight: bold;
+
+	&:hover {
+		transform: translateX(-5px);
+		filter: drop-shadow(1px 0px 0px black) drop-shadow(-2px 0px 0px black)
```

Comment:
on mobile the drop shadow looks buggy.

Reply:
What do you mean by buggy?

Reply:
the top shadow/border shows and disappears.

Reply:
if we are really short on time, we can just merge this not too big of a deal

Reply:
Yeah let's just merge.

Reply:
Not necessarily mobile but Safari/WebKit
<img width="258" alt="Screen shot of resource tag border glitching." src="https://github.com/HackAtUCI/zothacks-site-2023/assets/44419552/88baafcf-a8dd-4830-8590-15373880566f">

Reply:
Potential workaround, used before in Mentor section for `BookmarkLink`, is adding `transform: translateZ(0);` to the parent.

Reply:
Could this be fixed setting the `-webkit-filter` property to the same value?